rpc/src/endpoint/block.rs

```diff
@@ -48,3 +61,82 @@ pub struct Response {
 }
 
 impl crate::Response for Response {}
+
+pub mod v0_38 {
+    use std::vec::Vec;
+
+    use block::{Commit, Header};
+    use tendermint::evidence;
+
+    use tendermint_proto::v0_38::types::Block as RawBlock;
+
+    use super::*;
+
+    #[derive(Clone, Debug, Deserialize, Serialize)]
+    pub struct DialectResponse {
+        /// Block ID
+        pub block_id: block::Id,
+
+        /// Block data
+        pub block: DialectBlock,
+    }
+
+    impl crate::Response for DialectResponse {}
+
+    impl From<DialectResponse> for Response {
+        fn from(msg: DialectResponse) -> Self {
+            Self {
+                block_id: msg.block_id,
+                block: msg.block.into(),
+            }
+        }
+    }
+
+    #[derive(Clone, Debug, Deserialize, Serialize)]
+    #[serde(try_from = "RawBlock", into = "RawBlock")]
+    pub struct DialectBlock {
+        /// Block header
+        pub header: Header,
+
+        /// Transaction data
+        pub data: Vec<Vec<u8>>,
+
+        /// Evidence of malfeasance
+        // #[serde(with = "self::evidence_serializers")]
```

Comment:
Should this be removed?

Reply:
Yes since it does not have any impact given that we deserialize through RawBlock